### DISCUSSION TITLE
2006/12/04/%E4%BA%8B%E4%BB%B6%E7%BB%93%E6%9E%84

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#135] [Announcements]
# 2006/12/04/%E4%BA%8B%E4%BB%B6%E7%BB%93%E6%9E%84

阮奇桢

https://blog.qizhen.xyz/2006/12/04/%E4%BA%8B%E4%BB%B6%E7%BB%93%E6%9E%84

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Unknown
2008年10月2日 22:32 编辑
請問 7.4版要做到事件觸發怎麼做?有可能做嗎?
 
我現在需要進行一個程序
點一下按鈕然後進行一個附程式並記錄數據
可以指導一下嗎?

回复
Qizhen
2008年09月2日 08:55 编辑
更新了下载链接

回复
ttongg
2008年08月29日 09:29 编辑
范例不能下载,能发我邮箱里一份吗?
谢谢!
 
tongxiaozhao@126.com

回复
丽芸
2007年10月10日 17:04 编辑
我是一个LV的初学者,最近一直在看你的博,呵呵,很多不懂,以后请多指教:>

回复
玉文
2007年09月3日 21:37 编辑
…..oooO…………………..(……)…Oooo………….)…/. …(…..)…………..(_/…….)…/……………………. (_/……………. Pasé por.. …………………. Aquí ………………oooO…………………..(……)…Oooo………….)…/. …(…..)……………(_/…….)…/…………………….. (_/……………………………………..
很喜欢你的博客，希望以后可以看到更多好咚咚，祝开心哦！ 

回复
Qizhen
2007年01月26日 18:37 编辑
问题是这样的，我手上没有 LabVIEW 7.1 啊。

回复
jiyang
2006年12月11日 10:24 编辑
大x侠的文章对我们来说非常有用，可是我现在还在用labVIEW7.1，不能打开你给的例子，不知道以后能否也提供一下这个版本的例子：）

回复
波
2006年12月9日 14:07 编辑
我是个新手，看了你的blog真是受益匪浅。很希望能结识前辈:(
